Kuroba/app/src/main/java/com/github/adamantcheese/chan/core/site/sites/Wired7.java

```diff
@@ -82,6 +82,7 @@ public void setup() {
         setBoards(
                 Board.fromSiteNameCode(this, "Lewds & +18", "18"),
                 Board.fromSiteNameCode(this, "Random", "b"),
+                Board.fromSiteNameCode(this, "Flash", "f"),
```

Comment:
SWF files aren't supported. Basically nothing on this board will work. There's a reason 4chan's /f/ isn't supported.

Reply:
Ah, ok. I still think people would like to be able to browse the text itself though, but alright.